core/templates/pages/community-dashboard-page/community-dashboard-page.component.html

```diff
@@ -52,14 +52,14 @@
       </div>
       <div class="oppia-opportunities-tabs-explanation" data-tab-name="<[$ctrl.tabsDetails[$ctrl.activeTabName].tabName]>">
         <[$ctrl.tabsDetails[$ctrl.activeTabName].description]>
-      </div>
-      <div class="oppia-dashboard-language-container" ng-if="$ctrl.showLanguageSelector()">
-        <span class="oppia-dashboard-language-container-label">Translate text to:</span>
-        <select class="oppia-opportunity-language-selector protractor-test-language-selector"
-                ng-change="$ctrl.onChangeLanguage()"
-                ng-model="$ctrl.languageCode"
-                ng-options="language.id as language.description for language in $ctrl.languageCodesAndDescriptions">
-        </select>
+        <div class="oppia-dashboard-language-container" ng-if="$ctrl.showLanguageSelector()">
+          <span class="oppia-dashboard-language-container-label">Translate to</span>
+          <translation-language-select [options]="$ctrl.languageCodesAndDescriptions"
```

Comment:
Should be translation-language-selector perhaps, since that's more of a noun?

Reply:
Done